### DISCUSSION TITLE
blog/envoy-oss-5-year/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#48] [Announcements]
# blog/envoy-oss-5-year/

开源网络代理 Envoy 的创始人 Matt Klein，在 Twitter 因为自己的程序 bug 造成重大事故而离职，后加入 Lyft，在开源 Envoy 之前几乎没有贡献和管理开源项目的经验，这篇文章分享了他个人及 Envoy 开源的心路历程，在投身开源 Envoy 还是为雇主 Lyft 效命，该如何抉择？

https://lib.jimmysong.io/blog/envoy-oss-5-year/

**Comment by hedu00** [comment]
一款伟大产品的成长之路

**Reply by rootsongjc** [reply]
值得我们思考和借鉴。
